Proj0810: Remove unused package versions (#96)
dotnet-project-file-analyzers/dotnet-project-file-analyzers.github.io: rules/Proj0810.md

---
parent: Central Package Management
ancestor: Rules
---

# Proj0810: Remove unused package versions
When using [Central Package Management](Proj0800.md) the central

> `Directory.Packages.props` contains `<PackageVersion>`'s for all

Reviewer: ```suggestion
`Directory.Packages.props` contains `<PackageVersion>`s for all
```

> `<PackageRefence>`'s in the solution. To reduce noise, thie rule reports on

Reviewer: ```suggestion
`<PackageRefence>`s in the solution. To reduce noise, this rule reports on
```

> `<PackageVersion`'s that are (not longer) referenced. This rule requires a

Reviewer: ```suggestion
`<PackageVersion`s that are (no longer) referenced. This rule requires a
```
